utils/auth/jwtHelpers.ts

```diff
@@ -0,0 +1,97 @@
+import admin from 'firebase-admin';
+import { NextApiHandler, NextApiRequest, NextApiResponse } from 'next';
+
+export interface UserDetailsFromRequest {
+  uid: string;
+  isAnonymous: boolean;
+}
+
+// Note: do not modify the 'firebase-admin' default import style, it triggers a known bug
+// See: https://github.com/firebase/firebase-admin-node/issues/593
+try {
+  admin.initializeApp({
+    projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
+  });
+} catch (error) {
+  // Ignore the "already exists" message in dev environments (cause by hot reloads).
+  if (!(process.env.NODE_ENV == 'development' && /already exists/u.test('' + error))) {
+    console.error(error);
+  }
+}
+
+const UNABLE_TO_AUTHENTICATE = 'User could not be authenticated.';
+const ANONYMOUS_USER_NOT_AUTHORIZED = 'User has not verified their identity.';
```

Comment:
Nit, wonder if the option below is more concise, it also does not limit "not authorized" to anonymous users
```suggestion
const USER_NOT_AUTHORIZED = 'User has not verified their identity.';
```